i18n.go

```diff
@@ -192,11 +193,11 @@ func hasAcceptLanguageHeader(request *Request) (bool, string) {
 func hasLocaleCookie(request *Request) (bool, string) {
 	if request != nil && request.Cookies() != nil {
 		name := Config.StringDefault(localeCookieConfigKey, CookiePrefix+"_LANG")
-		if cookie, error := request.Cookie(name); error == nil {
+		cookie, err := request.Cookie(name)
+		if err == nil {
```

Comment:
That would keep the `err` variable from scope from the following `TRACE.Printf`. The dropping of the else is suggested by golint:

> if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)

including the moving of the variable declaration outside of the conditional statement